app/src/main/kotlin/com/wire/android/ui/common/CopyButton.kt

```diff
@@ -0,0 +1,35 @@
+package com.wire.android.ui.common
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.wire.android.R
+import com.wire.android.ui.common.button.IconAlignment
+import com.wire.android.ui.common.button.WireTertiaryButton
+import com.wire.android.ui.common.button.wireSecondaryButtonColors
+
+@Composable
+fun CopyButton(onCopyClicked: () -> Unit, modifier: Modifier = Modifier) {
+    WireTertiaryButton(
+        colors = wireSecondaryButtonColors(),
+        onClick = onCopyClicked,
+        leadingIcon = {
+            Icon(
+                painter = painterResource(id = R.drawable.ic_copy),
+                contentDescription = stringResource(R.string.content_description_right_arrow),
+                modifier = modifier
+                    .size(dimensions().conversationBottomSheetItemSize)
+            )
+        },
+        minHeight = 32.dp,
+        minWidth = 40.dp,
```

Comment:
Shouldn't these be in WireDimensions? 🤔

Reply:
resolved in https://github.com/wireapp/wire-android-reloaded/pull/453